microsvcs/gpt_generation/README.md

```diff
@@ -0,0 +1,85 @@
+# api quick reference sheet
+## POST /api/scenario
+*Debating on a GET? maybe with query instead?* 
+
+creates a new scenario for the user. 
+
+@params req.body.uid - required, user id, currently email <br>
+@params req.body.topic - optional, a selected or predefined topic to be used <br>
+
+Sample request body:
+```
+{
+    "uid":"sammyho@email.com", //required
+    "topic":"Diversity" //optional 
+}
+```
+
+Sample success response: 
+```
+{
```

Comment:
Not sure if you storing these scenarios.

Can consider generating a corresponding scenario ID